legal-api/src/legal_api/services/warnings/business/business_checks/involuntary_dissolution.py

```diff
@@ -0,0 +1,56 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Service to check involuntary dissolution for a business."""
+from legal_api.services.involuntary_dissolution import InvoluntaryDissolutionService
+
+from . import BusinessWarningCodes, WarningType
+
+
+def check_business(business: any) -> list:
```

Comment:
This isn't the legal name work where we had different business classes. 

It should be `def check_business(business: Business) -> list:`

Reply:
Updated